pyflagser/flagser.py

```diff
@@ -1,11 +1,127 @@
 """Implementation of the python API for the flagser C++ library."""
 
 import numpy as np
+
+from ._utils import _extract_static_weights, _extract_persistence_weights
 from flagser_pybind import compute_homology, implemented_filtrations
 
 
-def flagser(flag_matrix, min_dimension=0, max_dimension=np.inf, directed=True,
-            filtration="max", coeff=2, approximation=-1):
+def flagser_static(flag_matrix, min_dimension=0, max_dimension=np.inf,
+                   directed=True, filtration="max", coeff=2,
```

Comment:
Yes, I do.

Reply:
Thanks. Feel free to suggest an alternative. It's a bit tricky I think: I was thinking of different options ranging from the (unsatisfyingly non-descriptive but short) "algorithm" to the (maybe too wordy?) "completion_method".

Reply:
Ah, that's not what I meant. I think there is actually no need for a filtration because everything will appear at the same time as all edges have the same weight. The results are thus independent of the filtration algorithm. I could create a test to check that.

Reply:
I completely misunderstood what you said, sorry.

There are no class here, so static cannot be misinterpreted IMO. However, the difference between the static and persistence version lies also in the fact that the static expect an unweighted graph while the persistence expects a weighted one. We could call them `flagser_unweighted` and `flagser_weighted`.

Reply:
Ah, that's not what I meant. I think there is actually no need for a filtration because everything will appear at the same time as all edges have the same weight. The results are thus independent of the filtration algorithm. I could create a test to check that.

Reply:
I just saw the other discussion on that you started that was "hidden". I believe filtration should not be a parameter at all.

Also, I would like to suggest to change from `_static` to `_unweighted` and `_persistence` to `_weighted`. What do you think?

Reply:
> I believe filtration should not be a parameter at all.

I believe it too.

> Also, I would like to suggest to change from _static to _unweighted and _persistence to _weighted. What do you think?

I'd say this is a good idea. The nature of the input would be made utterly clear by this.

Reply:
Done.